project/build.scala

```diff
@@ -151,7 +151,14 @@ object build {
     }),
     scalacOptions ++= {
       CrossVersion.partialVersion(scalaVersion.value) match {
-        case Some((2, 11 | 12)) =>
+        case Some((2, 12)) =>
+          Seq(
+            "-Xsource:2.12",
+            "-opt:l:method,inline",
```

Comment:
pretty sure `method` is technically redundant but why the hell not

Reply:
nope, it's needed apparently. Although I can't validate that *anything* has changed.